### DISCUSSION TITLE
New QSO Area Layout

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#232] [Q&A]
Hi,

thank you very much for the new version 0.26.

However, with the new "New QSO Area Layout" I have the question why this was added and what is the idea / necessity behind it? 

I don't see a big benefit here, because the information is already included in the Classic-Layout and the arrangement in the Classic-Layout is actually useful and space-saving. 

From my point of view, it is good that it is possible to select the Classic layout and continue using it.

**Comment by foldynl** [comment]
The idea behind is:
1) I would like to add new fields to the New QSO widget. They will be optional but they can be present.
2) if someone doesn't like the order, then it can be changed. For example, QSO on 50MHz, a part of the QSO is a usually grid, so in this case I adjust the order to the grid, name QTH. On the other hand, locator is rarely present in case of  HF QSO.
3) If you don't want to have the Comment field then you can remove it and save more space.

**Comment by dl2ki** [comment]
OK, thanks for the info.
I will leave it as it is.
